pkgs/development/python-modules/nmcli/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "nmcli";
+  version = "1.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ushiboy";
+    repo = "nmcli";
+    tag = "v${version}";
+    hash = "sha256-1gVj4WfTx1NcoyWA9OK5EyGze9hmrXV0Mq50C1S3bfM=";
+  };
+
```

Comment:
You also seem to have forgotten the patching of the `nmcli` path here, you do need to add that back.